### ISSUE TITLE
Flexion worsens symptoms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9876]
**Preferred term label:**
Flexion worsens symptoms

**Synonyms**
Pain

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cubital tunnel syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034802
Exacerbated by flexion
Applied to a sign or symptom that is aggravated by (or made more painful by) bending of a limb or joint.
